patches/go-1.23/0002-Always-restrict-boringcrypto-crypto-tls-to-FIPS.patch

```diff
@@ -0,0 +1,42 @@
+From 5256a813afbb9c0f7d7ae00544ae1cbaeafb7f1e Mon Sep 17 00:00:00 2001
+From: Gavin Inglis <giinglis@amazon.com>
+Date: Tue, 13 Aug 2024 22:00:50 +0000
+Subject: [PATCH] Always restrict boringcrypto crypto/tls to FIPS
+
+Signed-off-by: Gavin Inglis <giinglis@amazon.com>
```

Comment:
I don't feel strongly about this, but for a patch that you didn't create but need to adjust context for an update, you can follow this convention from kernel development:
```suggestion
From 5256a813afbb9c0f7d7ae00544ae1cbaeafb7f1e Mon Sep 17 00:00:00 2001
From: Gavin Inglis <giinglis@amazon.com>
Date: Tue, 13 Aug 2024 22:00:50 +0000
Subject: [PATCH] Always restrict boringcrypto crypto/tls to FIPS

Signed-off-by: Ben Cressey <bcressey@amazon.com>
[giinglis: update for Go 1.23]
Signed-off-by: Gavin Inglis <giinglis@amazon.com>
```

Goal is to retain the original authorship but also take credit / blame for the changes you made.

Reply:
That's definitely a more appropriate outcome and reminiscent of coauthored / cherry picked commits. I'll adjust 👍